### ISSUE TITLE
DPI-awareness

### ISSUE THREAD

**Issue by ro-dev** [miranda-ng/miranda-ng#676]
What about to make Miranda NG a DPI-aware application?
Because now it's not being good looking when OS DPI settings set to 200%.

**Comment by georgehazan** [comment]
you mean contact list or any other part of Miranda?
what concretely went wrong?

**Comment by ro-dev** [comment]
Yes, contact list don't scale at all. I manage to scale up text size
set related options. But there is no options to make contact protocol icons big.

Miranda tray icon is pixelized (don't scale property).
Though, in Miranda Options => Icons => Status Icons => Global
similar icon looks good.

Text shift in follow windows (text should not cross the icon):
- Miranda NG options
- Login to Miranda NG
- Accounts

Example:
![miranda text shift 1](https://cloud.githubusercontent.com/assets/25161161/22185440/9240bcec-e0f6-11e6-8867-197f50fb76a0.PNG)

**Comment by eggsgent** [comment]
![mirandang_dpi_unaware](https://cloud.githubusercontent.com/assets/7951056/22862081/156fab82-f128-11e6-9c4e-e94d5a503418.png)
![mirandang_win10_125percent_scaling](https://cloud.githubusercontent.com/assets/7951056/22862083/1573ae76-f128-11e6-9675-bafd55b66115.png)
![mirandang_scriver-chat_window_win10_125procent-scaling](https://cloud.githubusercontent.com/assets/7951056/22862082/1571edf2-f128-11e6-846e-2e61db57e68b.png)

I switched from a 900p screen to a 1080p screen on my laptop and had to set Windows 10 scaling to 125%. Attached screenshots of blurry text and icons using development builds of MirandaNG. The system tray icons are also blurry)

Related info on Win10: https://blogs.windows.com/buildingapps/2016/10/24/high-dpi-scaling-improvements-for-desktop-applications-and-mixed-mode-dpi-scaling-in-the-windows-10-anniversary-update/#T8iE1IgcVPfe27Ki.97

**Comment by Revertron** [comment]
Yep, pixelized application in 2020 is very frustrating :(
